Helper/Requests.php

```diff
@@ -218,6 +214,17 @@ public function buildAddressData($billingAddress, $shippingAddress, $storeId, $r
                 $requestBilling["country"] = $billingAddress->getCountryId();
             }
 
+            if (!empty($billingAddress->getPostcode())) {
+                $requestBilling["postalCode"] = $billingAddress->getPostcode();
+                if ($billingAddress->getCountryId() == "BR") {
+                    $requestBilling["postalCode"] = preg_replace(
+                        '/[^\d]/',
+                        '',
+                        $billingAddress->getPostcode()
```

Comment:
The postcode is already in the var right?
```suggestion
                        $requestBilling["postalCode"]
```

Reply:
yes it is, we could go for your suggestion